velox/docs/monthly-updates/january-2022.rst

```diff
@@ -0,0 +1,28 @@
+********************
+January 2022 Update
+********************
+
+Core Library
+------------
+
```

Comment:
Should we also include the following?
- Add support for AS statement to PlanBuilder`
- Add support for AWS S3 IAM Roles

Reply:
Added. Thanks.